README.adoc

```diff
@@ -39,6 +39,22 @@ are supported.
 
 By default we include the URL for link:https://github.com/snowplow/iglu-central[Iglu Central], which includes schemas for the built-in tracking JSONs. You can add as many repositories as you like.
 
+==== Using the universal debugger
+
+The universal debugger uses a link:https://ngrok.com/[Ngrok] tunnel in order to process events. Ngrok provides a HTTP endpoint that we can use as the temporary collector endpoint.
+
+To start the tunnel ensure you have downloaded ngrok locally and create a new tunnel with:
+
+`./ngrok http example.snplow.net`
+
+Where example `example.snplow.net` is the host that you wish to forward requests on to. ngrok will expose a local API (on port 4040 by default) which the debugger will call to retrieve these events.
+
+Once your tunnel has started ngrok will return a HTTP and HTTPS url that you should then use as your collector host. Events sent to this host will be intercepted by ngrok and forwarded to the your Snowplow destination without modification.
```

Comment:
Not super familiar with ngrok, can you also configure it to not? Like a requestbin or something. That might also be handy/worth a mention if possible.

Reply:
Not as in configuring it to just capture the events but not forward them on at all? If so yeah, you can get Ngrok to just capture the events without the forwarding.

Reply:
Yeah that's what I meant. Neat!